### DISCUSSION TITLE
Skin files deobfuscation + new skinning elements for lazer

### DISCUSSION THREAD

**Discussion by Tenkotataii** [ppy/osu#31013] [Ideas]
I didn't see a proper idea for that, but I think there should be an option to deobfuscate the skin files to work directly through osu!lazer folders. It would help for mainly osu lazer players who want to edit their skin with a custom cursor, or a good sounding hitsound.
I would also suggest an addition of new skinning elements, for example, ur bar or text box.
And also... YOU SHOULD FIX THE SCOREBAR-BG IMPROPER SCALING, NOW!

**Comment by cl8n** [comment]
https://github.com/ppy/osu/pull/30226 probably addresses what you mean about "deobfuscation"
